### DISCUSSION TITLE
3.2 函数式编程

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#116] [Announcements]
# 3.2 函数式编程

SICP Python 中文翻译

https://composingprograms.netlify.app/3/2

**Comment by lyu427** [comment]
3.2.3 one-through-four 那一块返回的结果有缺失

**Reply by lyu427** [reply]
准确地说是后半部分的所有内容都缺少了返回结果

**Reply by clcs** [reply]
新版已增加可运行 scheme 代码块
